torch/csrc/jit/passes/create_functional_graphs.cpp

```diff
@@ -226,15 +226,17 @@ struct MutationRemover {
 
  private:
   bool newMemoryLocation(Value* v) {
-    // bail on nodes with side effects, blocks, or graphs
+    // bail on nodes with side effects, blocks, or graph / graph inputs
     Node* n = v->node();
     bool unhandled_node = n->blocks().size() != 0 ||
-        n->hasAttribute(attr::Subgraph) || n->hasSideEffects();
+        n->hasAttribute(attr::Subgraph) || n->hasSideEffects() ||
+        (v->node()->kind() == prim::Param);
```

Comment:
curious: why do we need to add a filter for params here, but we didn't need it before?